README.md

```diff
@@ -14,7 +14,7 @@ These code samples were created with the project templates available in Visual S
 
 ## Universal Windows Platform (UWP) samples/tools
 
-These samples require Visual Studio 2017 Update 4 or higher and the Windows Software Development Kit (SDK) version 16299 for Windows 10.
+These samples require Visual Studio 2017 Update 4 or higher and the Windows Software Development Kit (SDK) version 16299 for Windows 10 or higher on a per-sample basis.
```

Comment:
>16299 [](http://example.com/codeflow?start=115&length=5)

the sdk can still be that old for all the samples?

Reply:
yeah we can update base SDK in a separate PR, it involves more than a MD change